src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
We should probably change the name here. Not sure about the size constraint given our layout.